WordPress/src/main/java/org/wordpress/android/widgets/PostListButtonType.kt

```diff
@@ -31,6 +31,12 @@ enum class PostListButtonType constructor(
             R.attr.colorOnSurface
     ),
     BUTTON_CANCEL_PENDING_AUTO_UPLOAD(
+            14,
+            R.string.cancel,
+            R.drawable.ic_undo_white_24dp,
+            R.attr.wpColorWarningDark
+    ),
+    BUTTON_CANCEL_PENDING_AUTO_UPLOAD_COMPACT(
```

Comment:
We might want to consider refactoring `PostListButtonType` into a sealed class instead of introducing a new type

`data class BUTTON_CANCEL_PENDING_AUTO_UPLOAD(val isCompact: Boolean) : PostListButtonType(14, if (isCompact) R.string.cancel else R.string.pages_and_posts_cancel_auto_upload, R.drawable.ic_undo_white_24dp, R.attr.wpColorWarningDark)`

I'm not sure about this. I'd like to hear your thoughts. :)

Reply:
This sounds like a good idea!